src/badge/badge.resolver.ts

```diff
@@ -29,4 +29,10 @@ export class BadgeResolver {
   async badge(@Args('badgeId') id: string, @GqlUser('id') playerId: string): Promise<Badge[]> {
     return await this.badgeService.getBadge(id, playerId);
   }
+
+  @Query()
```

Comment:
Why this Query? We don't want to be able to grant rewards through API. That's internal logic.